packages/@aws-cdk/aws-ec2/lib/instance-types.ts

```diff
@@ -391,6 +391,7 @@ export enum InstanceSize {
   XLARGE = 'xlarge',
   XLARGE2 = '2xlarge',
   XLARGE4 = '4xlarge',
+  XLARGE6 = '6xlarge',
```

Comment:
you'll need to add a linter exception for this in [package.json](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ec2/package.json#L299-L300)

Reply:
why not add the public api instead? In fact, would be nice to do it for all instance types here and drop the linter exceptions.

Reply:
you mean link to the API docs? - sure, I think that's more helpful to users. 
Not sure that all instance types need to be done in this PR given it's intended scope was to add the missing instance type. I'm all for removing linter exceptions. This module has the largest list of exceptions in the entire CDK (most of them the variations of windows versions)

Reply:
>I think that's more helpful to users
I would usually agree, but in this case, there isn't any real valuable docs to add to each instance size?

Reply:
Added doc to all InstanceSize